integrations/cyberchef/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine3.10
```

Comment:
please cite where you copied this: https://github.com/gchq/CyberChef-server/blob/master/Dockerfile